### DISCUSSION TITLE
What branch supports v2.0.1?

### DISCUSSION THREAD

**Discussion by mc12345678** [zencart/zencart#6610] [Q&A]
Where does the github branch for v2.0.1 end and v2.1.0 begin? Certainly, there are corrections posted/made specific for v2.0.1 that eventually are applicable also to v2.1.0, but how does one now use Zen Cart github repo to figure this out? There is no v201 branch in the way there are v158, v157, v156, etc... branches.

**Comment by lat9** [comment]
https://github.com/zencart/zencart/tree/v2.0.1 contains the files shipped for v2.0.1

**Reply by mc12345678** [reply]
> https://github.com/zencart/zencart/tree/v2.0.1 contains the files shipped for v2.0.1

You know that's a tag not a branch. 

As stated, that tree marker contains the files "shipped for" (distributed as) v2.0.1, but does not include improvements/corrections made following that release specifically about issues existing in that release.

The next branch in line is master which is already reflecting v2.1.0 alpha... 

Already there have been statements made about how only the truly daring or knowledgeable should implement an unreleased version. 

If it were my own site, I wouldn't have a problem with it and would be willing to devote the extra time and attention. I would rather install already corrected software.

**Comment by scottcwilson** [comment]
Everything that's in 2.0.1 should also be in 2.1.0.  If you have found something that has been missed, please post specifics.

**Comment by mc12345678** [comment]
Based on the above two responses, seems to me the answer to this question is as follows:

Zen Cart code managers no longer maintain a version specific branch associated with the latest release to capture associated solutions while continuing to prepare for the next version.

> Everything that's in 2.0.1 should also be in 2.1.0. If you have found something that has been missed, please post specifics.

This was never in question (even though such misses have historically occurred), but affirms there is no such branch.

As such, there are likely some sort of solutions in the next main version release that can be applied to the previous version to maintain operation.

Examples of apparent code changes to correct v2.0.1 issues (not incorporated into a v2.0.1 branch) that are in v2.1.0: 56665f8e45d46dbc503b73f9d1e9a768990448fa, 18511c40a64c76f0931fd82af42f56d6a6c5772a, fa3d05140563da5578925634919980f51faea2b4.

**Comment by scottcwilson** [comment]
- We're trying to not have to actively maintain multiple branches at the same time.  
- The 2.0 branch (which was used to create 2.0.1) still exists.  We are not actively backporting to it in the hope that 2.1.0 will be out soon. 
- In the event of discovery of a serious bug in 2.0.1, we could certainly build a 2.0.2.  
- The bugfix commits you reference above have PRs that are tagged "2.0.2."  This tag means, "if we do a 2.0.2 build, pull this fix in." 
- Some of this is explained in https://github.com/zencart/zencart/discussions/6502
